### PR TITLE
Default Website

### LINKED ISSUE #571: Resources: apache2_default_website
### Cookbook version
- 6.0.0-beta

### Chef-client version
- Latest

### Platform Details
- All supported

### Scenario:
As a user
I want to be able to enable and disable the default website
So that I do not serve content from it

As a user
I want to be able to configure the default website
So that I can serve simple content from it

### Acceptance Criteria:
- Apache default website should

### Actual Result:
Scenario disable website
- [ ] Unit test to make sure the default website content is not on disk.
- [ ] Inspec test to make sure that when a user goes to the the default website, it doesn't respond

Scenario configure website
- [ ] Unit test to make sure the content is present
- [ ] Unit test to configure and check content on disk
- [ ] Inspec test (new kitchen suite) to check the website responds

### PR DESCRIPTION
**Pull request by damacus**
## Description

Add default_website resource

### Issues Resolved

#571 

### Check List
- [ ] All tests pass. See https://github.com/sous-chefs/apache2/blob/master/TESTING.md
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable

### CODE REVIEW

**Comment by kitchen-porter** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          2 Errors
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">Please add labels to this Pull Request</td>
    </tr>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">Please include a <a href="https://github.com/sous-chefs/line-cookbook/blob/master/CHANGELOG.md">CHANGELOG</a> entry.</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This is a big Pull Request.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: README.md [hunk 1]
```diff
@@ -1,29 +1,29 @@
 # apache2 Cookbook
-[![Cookbook Version](https://img.shields.io/cookbook/v/apache2.svg?style=flat)](https://supermarket.chef.io/cookbooks/apache2) [![Build Status](https://travis-ci.org/sous-chefs/apache2.svg?branch=master)](https://travis-ci.org/sous-chefs/apache2) [![License](https://img.shields.io/badge/license-apache2_2-blue.svg)](https://www.apache.org/licenses/LICENSE-2.0)
+[![Cookbook Version](https://img.shields.io/cookbook/v/apache2.svg?style=flat)](https://supermarket.chef.io/cookbooks/apache2) [![License](https://img.shields.io/badge/license-apache2_2-blue.svg)](https://www.apache.org/licenses/LICENSE-2.0)
```
Comment (Xorima) [inline README.md]: We should add in the circleci build badge?
```suggestion
[![Cookbook Version](https://img.shields.io/cookbook/v/apache2.svg?style=flat)](https://supermarket.chef.io/cookbooks/apache2) [![License](https://img.shields.io/badge/license-apache2_2-blue.svg)](https://www.apache.org/licenses/LICENSE-2.0)[![CircleCI](https://circleci.com/gh/sous-chefs/apache2.svg?style=svg)](https://circleci.com/gh/sous-chefs/apache2)
```

File: documentation/resource_apache2_config.md [hunk 2]
```diff
@@ -0,0 +1 @@
+# config
```
Comment (Xorima) [inline documentation/resource_apache2_config.md]: Should we have a bit more documentation here?

File: documentation/resource_apache2_install.md [hunk 3]
```diff
@@ -0,0 +1 @@
+# install
```
Comment (Xorima) [inline documentation/resource_apache2_install.md]: Should we have a bit more documentation here?

File: documentation/resource_apache2_default_site.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
+# default_site
+
+Controls the default site.
+
+```ruby
+apache2_default_site '' do
+  default_site_name String
+  site_action       [String, Symbol]
+  port              String
+  cookbook          String
```
Comment (Xorima) [inline documentation/resource_apache2_default_site.md]: Should we add an example of using a different cookbook to source the file from?

File: documentation/resource_apache2_config.md (hunk 2 above)
Comment (damacus) [reply documentation/resource_apache2_config.md]: Yep but that's for #572

**Comment by Xorima** [comment]
Should we have a changelog entry?

File: README.md (hunk 1 above)
Comment (damacus) [reply README.md]: Yeah, also leaving that for #572

**Comment by damacus** [comment]
Yeah totally, adding that changelog now :)

**Pull request merged**